data-exploration/patients-discharge-2.R

```diff
@@ -338,15 +368,25 @@ weekly_palliative <- weekly_palliative[,c("Year.of.Discharge", "Week.of.Discharg
 # arima
 weekly.palliative.model <- arima(weekly_palliative$weekly_count, order = c(1,0,0))
 weekly.palliative.prediction <-predict(weekly.palliative.model,n.ahead=prediction.length)
-weekly.palliative.admissions.pred <- weekly.palliative.prediction$pred[1:prediction.length]
-plot.ts(c(weekly_palliative$weekly_count, weekly.palliative.admissions.pred))
+weekly.palliative.discharges.pred <- weekly.palliative.prediction$pred[1:prediction.length]
+plot.ts(c(weekly_palliative$weekly_count, weekly.palliative.discharges.pred))
+# Add confidence interval (95%)
+weekly_palliative_pred_with_ci <- forecast.Arima(weekly.palliative.model, 20) %>%
+                                  as.data.frame()
+colnames(weekly_palliative_pred_with_ci) <- c("weekly_count", "low_80", "high_80", "lower (95%)", "upper (95%)")
+weekly_palliative_pred_with_ci <- weekly_palliative_pred_with_ci[,c("weekly_count", "lower (95%)", "upper (95%)")]
 # create df
 disch_palliative_pred <-mutate(data.frame(Year.of.Discharge = rep(last.year,prediction.length),
                                       Week.of.Discharge = (last.week+1):(last.week+prediction.length),
-                                      weekly_count = weekly.palliative.admissions.pred),
+                                      weekly_count = weekly.palliative.discharges.pred),
                                 weekly_count = weekly_count,
                                 Week.of.Discharge = sprintf("%02d", Week.of.Discharge),
                                 Year.of.Discharge = as.character(Year.of.Discharge))
+disch_palliative_pred_ci <-mutate(data.frame(Year.of.Discharge = rep(last.year,prediction.length),
```

Comment:
I wasn't aware there was any scaling going here. 
I thought this step was binding a data frame with week and year of discharge and a data frame with projections and CI.